Binary files gpm-1.20.5.orig/.Makefile.in.swp and gpm-1.20.5/.Makefile.in.swp differ
diff -pruN gpm-1.20.5.orig/src/lib/report-lib.c gpm-1.20.5/src/lib/report-lib.c
--- gpm-1.20.5.orig/src/lib/report-lib.c	2008-06-17 09:54:54.811257867 +1000
+++ gpm-1.20.5/src/lib/report-lib.c	2008-06-17 10:08:36.780265133 +1000
@@ -29,6 +29,10 @@ void gpm_report(int line, char *file, in
    char *string = NULL;
    int log_level;
    va_list ap;
+#ifndef DEBUG_VERBOSE
+   if(stat==GPM_STAT_DEBUG)
+   return;
+#endif
 
    if (stat == GPM_STAT_DEBUG) return;
 

